src/js/facility-locator/components/AppointmentInfo.jsx

```diff
@@ -26,7 +26,9 @@ export default class AppointmentInfo extends Component {
 
     const healthAccessAttrs = facility.attributes.access.health;
     const specialtyKeys = healthAccessAttrs && Object.keys(healthAccessAttrs);
-    delete specialtyKeys.primaryCare;
+    pull(specialtyKeys, 'primaryCare');
+    pull(specialtyKeys, 'effectiveDate');
```

Comment:
Looks like `pull` can pass multiple arguments for elements to remove. https://lodash.com/docs/4.17.4#pull

```
pull(specialtyKeys, 'primaryCare', 'effectiveDate');
```

@bshyong: There's a merge conflict with these changes on one of your PRs if you want to make the change there.